ci/format.sh

```diff
@@ -42,7 +42,9 @@ fi;
 
 BASE_SHA="$(git fetch $UPSTREAM master > /dev/null 2>&1 && \
            (git merge-base --fork-point FETCH_HEAD HEAD || git merge-base FETCH_HEAD HEAD))"
+set -f
```

Comment:
Can you put a comment here about why this is done, since it's non-obvious to someone without a lot of bash script experience?